autofill/autofill-internal/src/main/java/com/duckduckgo/autofill/internal/AutofillInternalSettingsActivity.kt

```diff
@@ -399,34 +399,48 @@ class AutofillInternalSettingsActivity : DuckDuckGoActivity() {
         binding.addSampleLoginsButton.setClickListener {
             val timestamp = dateFormatter.format(System.currentTimeMillis())
             lifecycleScope.launch(dispatchers.io()) {
-                sampleCredentials(domain = "fill.dev", username = "alice-$timestamp", password = "alice-$timestamp").save()
-                sampleCredentials(domain = "fill.dev", username = "bob-$timestamp", password = "bob-$timestamp").save()
-                sampleCredentials(domain = "subdomain1.fill.dev", username = "charlie-$timestamp", password = "charlie-$timestamp").save()
-                sampleCredentials(domain = "subdomain2.fill.dev", username = "daniel-$timestamp", password = "daniel-$timestamp").save()
+                listOf(
```

Comment:
these changes are just to make the dev settings screen more efficient at bulk insertion/deletion